Fastlane/Fastfile

```diff
@@ -134,7 +134,9 @@ lane :uploadScreenshots do
 end
 
 lane :tests do
-  run_tests(scheme: "MovappUITests")
+    run_tests(
+        scheme: "Movapp (iOS)"
```

Comment:
To faktu spustí testy v MovappUITests? :)

Reply:
Pustí to všechny testy navázané na to schéma, jelikož tam jsou všechny kromě screenshotů tak ano. Je to hezky vidět ve výstupu pipeline.